src/code42cli/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.7.3"
+__version__ = "0.7.4"
```

Comment:
Should we add an empty changelog?

Reply:
We'll add that entry to master separately since don't intend to merge this there.